app/preferences.go

```diff
@@ -14,21 +14,37 @@ import (
 type preferences struct {
 	*internal.InMemoryPreferences
 
-	prefLock     sync.RWMutex
-	ignoreChange bool
+	prefLock sync.RWMutex
+	// Normally, any update of preferences is immediately written to disk,
```

Comment:
We don't really go for large comments in code, in fact any code comments normally indicate that the code is not clear.
We have version control to learn more about the change if looking up the history is required.
In this case the variable names seem clear, can we remove the comments?